keyboards/kiserdesigns/madeline/keymaps/default/config.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2021 Noah Kiser (NCKiser)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#define TAP_CODE_DELAY 10
```

Comment:
this could be moved to qmk.tap_keycode_delay in the info.json file

Reply:
Would this be in a keymap-level info.json? The delay preferred is highly subjective, and would change based on the keymap used.